### PR TITLE
Add draft-ietf-httbis-h3-websockets.md to the repo

### PR DESCRIPTION
**Pull request by RyanTheOptimist**
Adds draft-ietf-httpbis-h3-websockets.md to the repo which is clone of [draft-hamilton-httpbis-h3-websockets.md](https://github.com/RyanTheOptimist/httpbis-h3-websockets/blob/main/draft-hamilton-httpbis-h3-websockets.md) with the "note to readers" added.

### CODE REVIEW

**Comment by LPardue** [comment]
Missing a p in the filename

File: draft-ietf-httbis-h3-websockets.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: "Bootstrapping WebSockets with HTTP/3"
+docname: draft-ietf-httbis-h3-websockets.md
+category: info
+
+ipr: trust200902
+area: TODO
```
Comment (mnot) [inline draft-ietf-httbis-h3-websockets.md]:
```suggestion
area: ART
```

File: draft-ietf-httpbis-h3-websockets.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+---
+title: "Bootstrapping WebSockets with HTTP/3"
+docname: draft-ietf-httbis-h3-websockets.md
+category: info
+
+ipr: trust200902
+area: TODO
```
Comment (mnot) [inline draft-ietf-httpbis-h3-websockets.md:7]:
```suggestion
area: ART
```

**Comment by RyanTheOptimist** [comment]
> Missing a p in the filename

That is embarrassing! Fixed. (In the PR description as well)

File: draft-ietf-httpbis-h3-websockets.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+workgroup: HTTP
+keyword: Internet-Draft
+
```
Comment (mnot) [inline draft-ietf-httpbis-h3-websockets.md:10]:
```suggestion
github-issue-label: h3-websockets
```

**Review by mnot — APPROVED** [review]
Looks good, two small things.

**Review by tfpauly — APPROVED** [review]

File: draft-ietf-httbis-h3-websockets.md (hunk 1 above)
Comment (RyanTheOptimist) [reply draft-ietf-httbis-h3-websockets.md]: Thanks! I applied the change to the renamed file.

**Pull request merged**